Code/Framework/AzCore/AzCore/Serialization/EditContext.cpp

```diff
@@ -8,6 +8,146 @@
 
 #include <AzCore/Serialization/EditContext.h>
 
+#include <AzCore/Console/IConsole.h>
+#include <AzCore/Console/IConsoleTypes.h>
+
+namespace AZ
+{
+    static void LogAssociativeContainerFields(const EditContext& editContext, const AZ::ConsoleCommandContainer&)
+    {
+        AZStd::string logString = "\nParent Reflected Class, Associative Element Name, Associative Type\n";
+        Edit::TypeVisitor logFieldVisitor;
+        logFieldVisitor.m_classDataVisitor = [&logString, &editContext](const Edit::ClassData& classData) -> bool
```

Comment:
It doesn't look like we modify `editContext` in the visitor method, but I suppose passing by reference is preferred here since it would be more efficient?

Reply:
Well the editContext isn't copyable, so it can only be passed by reference, plus we don't want to copy the EditContext anyway.